configuration/index.md

```diff
@@ -160,7 +160,7 @@ Please be aware that not every addon/binding of openHAB version 1.x has been upd
 You can still run these but you need to use file configuration for them.
 
 Things and Items can either be defined and managed in configuration files or handled by Paper UI in a [system-side database](/docs/administration/jsondb.html).
-For Things and Items, both methods can be used **side-by-side**, e.g. a Thing can be discovered and created by Paper UI and the Items that link to that very same Thing (that Thing's Channels, actually) can be manually defined in a `.items` configuration file. 
+You can use PaperUI for Things and files for Items, e.g. a Thing can be discovered and created by Paper UI and the Items that link to that very same Thing (that Thing's Channels, actually) can be manually defined in a `.items` configuration file. It would work to but you **should not** use both methods in parallel. You will easily forget about what is your "source of truth" when you reconfigure Things or Items at a later stage if that's different per-Item or per-Thing, so do yourself the favor and use _either_ method but not both.
```

Comment:
New discussion path since the code changed.

Here is my suggestion:

```suggestion
Both of the above described methods could be used parallel, e.g. a Thing can be discovered and created by Paper UI and the Items that link to that very same Thing (that Thing's Channels, actually) can be manually defined in a `.items` configuration file.

> Alltough it's technically possible to use both of methods parallel, we recommend to choose only one of them to avoid confusion and a loss of overview of your openHAB environment.
> It would work to but you may easily forget about what is your "source of truth" when you reconfigure Things or Items at a later stage.
```

I have added a blockquote, which will highlight the last part in the generated page.

And this is now valid with our **one line per sentence** guideline, like Scott already mentioned above.